packages/components/src/components/multi-select/multi-select/multi-select.tsx

```diff
@@ -166,40 +167,50 @@ export class MultiSelect {
 
   @Watch('value')
   public onValueChange(): void {
+    this.setFormValue(this.value);
     // When setting initial value the watcher gets called before the options are defined
     if (this.multiSelectOptions.length > 0) {
       if (!this.preventOptionUpdate) {
         setSelectedOptions(this.multiSelectOptions, this.value);
       }
       this.preventOptionUpdate = false;
-      if (this.isWithinForm) {
-        updateNativeOptions(this.nativeSelect, this.multiSelectOptions);
+    }
+  }
+
+  @Watch('form')
+  public updateFormAssociation(): void {
+    if (this.form) {
+      const formElement = document.getElementById(this.form) as HTMLFormElement;
+      if (formElement) {
+        formElement.appendChild(this.host);
```

Comment:
Is this correct? Doesn't seem like a good idea to change the DOM here. Isn't there a way to associate with a new form?

Reply:
After testing it doesn't seem to be necessary. Not sure under which circumstances I added it 🤔. I tested it outside the form and it works just fine, also the e2e test succeeds, so I removed it.  Same for select